viYarn/Episode/model/Author.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Author.swift
+//  viYarn
+//
+//  Created by vieon on 22.09.2018.
+//  Copyright © 2018 RIEON. All rights reserved.
+//
+
+import Foundation
+
+struct Author {
+    let id: Int
+    let slug: String
+    let last_name: String
+    let first_name: String
+    let name: String
+    let nickname: String
+    let url: String
+    let description: String
+}
+
+extension Author {
+    init?(json: [String: Any]) {
+        guard let id = json["id"] as? Int else { return nil }
+        guard let slug = json["slug"] as? String else { return nil }
+        guard let last_name = json["last_name"] as? String else { return nil }
```

Comment:
p0: so this should be
guard let lastName = json["last_name"] as? String else { return nil }
guard let firstName = json["first_name"] as? String else { return nil }